manifests/init.pp

```diff
@@ -19,7 +19,9 @@
   $option_static_route = undef,
   $options            = undef,
   $authoritative      = false,
-  $dhcp_root_group    = $dhcp::params::root_group
+  $dhcp_root_group    = $dhcp::params::root_group,
+  $pools              = {},
+  $hosts              = {}
```

Comment:
Nowadays we add trailing commas so I know it wasn't there before, but mind adding it while you're here?